actions/setvar.go

```diff
@@ -92,15 +96,23 @@ func (a *setvarFn) evaluateTxCollection(r rules.RuleMetadata, tx rules.Transacti
 		if len(value) > 1 {
 			sum, err = strconv.Atoi(value[1:])
 			if err != nil {
-				tx.DebugLogger().Error("[%s] Invalid value for setvar %q on rule %d", tx.ID(), value, r.ID())
+				tx.DebugLogger().Error().
```

Comment:
I think all the logs should have a Msg, eg invalid value for this one

Reply:
Done